app.go

```diff
@@ -104,9 +138,11 @@ func (app *App) processEvent(event *broker.Event) *string {
 		return nil
 	}
 
-	txOpts := &eos.TxOptions{}
+	var txOpts *eos.TxOptions
 	err := utils.RetryWithTimeout(func() error {
-		return txOpts.FillFromChain(api)
+		var e error
+		txOpts, e = app.getTxOpts()
```

Comment:
you can just write `txOpts, e := app.getTxOpts()` and save a line of code 😉

Reply:
here we can't do that, because *txOpts is already defined outside current scope, so if we would use ":=" then it create new local `txOpts` variable

Reply:
oh, yes you're right